Azure-Samples/azure-search-openai-demo
# Clarify which search Skus are compatible

## Purpose

Related to: https://github.com/Azure-Samples/azure-search-openai-demo/issues/586
This change clarifies the free sku isnt compatible. 

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* azd up with a free sku service, see:

<img width="1347" alt="Screenshot 2023-09-17 at 7 58 00 AM" src="https://github.com/Azure-Samples/azure-search-openai-demo/assets/297042/71cd6111-8746-4ba6-813a-78494970379f">

* azd up with standard, still works
